lib/components/narrative/connected-trip-details.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux'
+import styled from 'styled-components'
+import TripDetailsBase from '@opentripplanner/trip-details'
```

Comment:
Wondering if there's a better name for this import. `UnstyledTripDetails`? I'm fine with what you have, but it does seem like there might be a better name out there.

Reply:
I'm not super opinionated about this. When I do this I usually do this the other way around a la:

```js
import TripDetails from  '@opentripplanner/trip-details'
import styled from 'styled-components'

const StyledTripDetails = styled(TripDetails)`
...
```

Reply:
Would be nice to agree on some kind of convention.